test/ffmpeg-vaapi/vpp/deinterlace.py

```diff
@@ -8,35 +8,79 @@
 from ..util import *
 
 spec = load_test_spec("vpp", "deinterlace")
+spec_r2r = load_test_spec("vpp", "deinterlace", "r2r")
```

Comment:
it use md5 for deinterlace test standard in our script, may be we do not need enable it at now

Reply:
@uartie , is it ok to keep R2R tests for deinterlace or remove it?

Reply:
I don't see anything wrong with keeping it

Reply:
@wangzj0601 can you elaborate why you think r2r is not useful now?

Reply:
Because we use md5 as standrad for deinterlace test, this can ensure that the md5 must be same every time, I think it is not necessary to enable it at now.

Reply:
MD5 is only a default metric in the test_default deinterlace test (i.e. non-r2r).  User configuration can change it to something else if they want.  Thus an R2R test may still be useful to users in some cases.  If we are using MD5 metric for internal config, then it is probably redundant to define any cases for R2R.  Since the test is already written, I don't see any harm in keeping it since user config does not have to enable it.